src/main/java/raisetech/student_management/controller/StudentController.java

```diff
@@ -42,77 +41,151 @@ public StudentController(StudentService service) {
   }
 
   /**
-   * 受講生詳細の一覧検索です。
-   * 全件検索を行うので、条件指定は行いません。
+   * 受講生詳細の一覧検索です。 全件検索や条件を指定することによって条件検索が可能です。
    *
-   * @return　受講生詳細一覧（全体）
+   * @return　受講生詳細一覧
    */
-  @Operation(summary = "一覧検索", description = "受講生の一覧を検索します")
+  @Operation(summary = "受講生詳細の一覧検索", description = "受講生詳細の一覧を検索します。条件をクエリパラメータに入力すると絞り込み検索が可能です")
   @GetMapping("/studentList")
-  public List<StudentDetail> getStudentList() {
-    return service.searchStudentList();
+  public List<StudentDetail> getStudentList(
+      @RequestParam(required = false)
+      String id,
+      @RequestParam(required = false)
+      String fullName,
+      @RequestParam(required = false)
+      String furigana,
+      @RequestParam(required = false)
+      String handleName,
+      @RequestParam(required = false)
+      String mailAddress,
+      @RequestParam(required = false)
+      String area,
+      @RequestParam(required = false)
+      Integer age,
+      @RequestParam(required = false)
+      String gender
+  ) {
+    Student student= new Student();
```

Comment:
改めて全体的にコードフォーマットできているか確認しましょう！
```suggestion
    Student student = new Student();
```

Reply:
手動じゃなくてフォーマッターで行いましょう